.circleci/config.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       - checkout
       - restore_cache:
           key: sbtcache
-      - run: sbt ++3.3.0 core/test catsInterop/compile benchmarks/compile monixInterop/compile clientJVM/test clientJS/compile zioHttp/test tapirInterop/test http4s/test federation/test tools/test reporting/test tracing/test
+      - run: sbt ++3.3.0 core/test catsInterop/compile benchmarks/compile monixInterop/compile clientJVM/test clientJS/compile zioHttp/test tapirInterop/test pekkoHttp/test http4s/test federation/test tools/test reporting/test tracing/test
```

Comment:
We need to add the `pekkoHttp` to the `test_mima` job(s) as well

Reply:
added a line here, but it fails. I guess since this is since there is nothing to compare to. should this be a separate PR later once 1 published version exists?

Reply:
Ah probably yeah

Reply:
Maybe just comment it out so it will be easy to add later